src/test/enums/http-method.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export enum HttpMethod {
```

Comment:
- I thinks its better to move under shared/ folder 
- Rename to `RequestType` just like it is named under `cypress/` folder to keep it consistent with Application.

I just also noticed that we have different name too + declared twice inside `cypress/` folder but IMO, it should be separate for the time being.

```ts
export enum RequestType {
  GET = 'GET',
 ...
}
```